### PR TITLE
Advertise infarm as a company which is using django on production.

### PR DESCRIPTION
**Pull request by ticosax**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -535,6 +535,10 @@ Something is missing?  Create a pull request!
         [github](https://github.com/incapptic)
         ,
         [jobs](https://www.incapptic.com/company-careers/)
+    * [Infarm - Indoor Urban Farming GmbH](https://infarm.com/)
+        [github](https://github.com/infarm/)
```
Comment (hartwork) [inline README.md]: - I tried to find use of Django at…

  - https://github.com/infarm/?language=python and
  - https://www.wappalyzer.com/lookup/infarm.com/

  …but didn't find any evidence myself.
  Please add a link for proof with use of Django visible to the public internet as all the other companies in this list have. E.g. a link to job ad mentioning Django (outside of "Laravel or Django or RoR") or a link to Django admin or `staticfiles.json` would do.
- An emdesh is missing between the company name and the link list that follows.
- https://www.infarm.com/en/imprint reads "InFarm – Indoor Urban Farming GmbH" with capital "F". is capital "F" or lower "f" correct? 

Thanks! :pray:

**Review by hartwork — CHANGES_REQUESTED** [review]
Hi @ticosax cool to hear! Some details need fixing though:

File: README.md (hunk 1 above)
Comment (ticosax) [reply README.md]: thanks for your feedback. I think I addressed the points.

Comment (hartwork) [reply README.md]: Excellent — thank you! :pray:

**Review by hartwork — APPROVED** [review]

**Pull request merged**